wpeverest/user-registration
# UR-1430 Enhancement - Screenshot video on upgrade to pro popup.

### All Submissions:






### Changes proposed in this Pull Request:



Previously, video of any pro field is not shown in the upgrade to pro popup when click there. This PR adds enhances the upgrade to pro popup.

### How to test the changes in this Pull Request:

1. Activate the user registration plugin.
2. Click on Add New menu. 
3. In form builder page, click on Pro field check whether video is shown or not in the upgrade to pro popup.

### Types of changes:




### Other information:




### Changelog entry

>Enhancement - Screenshot video on upgrade to pro popup.
